mapdata/WarcraftLegacies/BuffData/Skin/1211379778.json

```diff
@@ -0,0 +1,29 @@
+{
+  "OldId": 1835619650,
+  "NewId": 1211379778,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1885959270,
+      "Type": 3,
+      "Value": "Stunned"
+    },
+    {
+      "Id": 1700951398,
+      "Type": 3,
+      "Value": "This unit has been stunned. It cannot move, attack or cast spells."
```

Comment:
I meant that you don't need a custom buff at all. Just the normal Stunned buff is fine, i.e. the buff with the id BSTN.

Reply:
As you informed me on Discord, you do actually need a custom buff to get the water effect. Just rename this one to Stunned so it's otherwise identical, and I think it's sorted.